### PR TITLE
Add draft spec

### PR DESCRIPTION
**Pull request by NomeQ**
All content from original [GTFS-Flex doc](https://docs.google.com/document/d/1UTcpMJlANSoJ1ZEk5IrQh_plza1ZnvgwraMEBI_o2mw/edit) added.

### CODE REVIEW

**Comment by barbeau** [comment]
Awesome, thanks @NomeQ!  Could you please add a `CONTRIBUTORS` file to the root, and add Brian Ferris, @antrim, and yourself?

Something like what Google has for the [transit](https://github.com/google/transit/blob/master/CONTRIBUTORS) repo - maybe:

~~~
# People who have contributed content to this repository
# Names should be added to this file as:
#     Name <email address>
Brian Ferris <bdferris@google.com>
...
~~~

We don't have a CLA process here, but I'd like to have some record of who has edited content.  We could use Git commit history for this as well, but that would leave out Brian.

File: gtfs-flex/spec-reference.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+##Flexible Public Transportation Services in GTFS
```
Comment (barbeau) [inline gtfs-flex/spec-reference.md]: To following the existing [Google transit repo format](https://github.com/google/transit), could you please move and rename this file to `/gtfs-flex/spec/en/reference.md`?

**Comment by antrim** [comment]
@barbeau : I have responded to those comments.

**Comment by barbeau** [comment]
Thanks @antrim!

**Pull request merged**